zeppelin-interpreter/pom.xml

```diff
@@ -242,6 +249,29 @@
           <dateFormat>yyyy-MM-dd HH:mm:ss</dateFormat>
         </configuration>
       </plugin>
+      <plugin>
+        <groupId>org.antlr</groupId>
+        <artifactId>antlr4-maven-plugin</artifactId>
+        <version>${antlr4.version}</version>
+        <configuration>
```

Comment:
Please do not write the code to `src/main`.
A package does not need to be specified either.
The default location `${project.build.directory}/generated-sources/antlr4` is perfect.
I think the configuration block is unnecessary.